os/include/tinyara/binary_manager.h

```diff
@@ -159,6 +143,14 @@ struct kernel_binary_header_s {
 } __attribute__((__packed__));
 typedef struct kernel_binary_header_s kernel_binary_header_t;
 
+struct common_binary_header_s {
+	uint32_t crc_hash;
+	uint16_t header_size;
+	uint32_t version;
+	uint32_t binary_size;
```

Comment:
Can common binary be compressed?

Reply:
The compression is not supported for common binary now.